documentation/modules/exploit/multi/misc/consul_service_exec.md

```diff
@@ -0,0 +1,116 @@
+## Vulnerable Application
+
```

Comment:
List the vulnerable application and versions affected here.

Reply:
See https://github.com/rapid7/metasploit-framework/pull/10443/commits/46ee2a6c9484368bee752828b9ba7fd3eb5043bf